.storybook/pages/Cad/Cad.tsx

```diff
@@ -86,98 +210,130 @@ export const Cad = () => (
-                                as="h3"
-                                className="u-margin-bottom-sm"
-                                size="body-sm"
-                              >
-                                Project name
-                              </Heading>
-                              <div className="fpo u-margin-none">12 days</div>
-                            </CardBody>
-                          </Card>
-                        </GridItem>
-                        <GridItem>
-                          <Card>
-                            <CardBody>
-                              <Heading
-                                as="h3"
-                                className="u-margin-bottom-sm"
-                                size="body-sm"
-                              >
-                                Project name
-                              </Heading>
-                              <div className="fpo u-margin-none">12 days</div>
-                            </CardBody>
-                          </Card>
-                        </GridItem>
-                        <GridItem>
-                          <Card>
-                            <CardBody>
-                              <Heading
-                                as="h3"
-                                className="u-margin-bottom-sm"
-                                size="body-sm"
-                              >
-                                Project name
-                              </Heading>
-                              <div className="fpo u-margin-none">12 days</div>
-                            </CardBody>
-                          </Card>
-                        </GridItem>
-                        <GridItem>
-                          <Card>
-                            <CardBody>
-                              <Heading
-                                as="h3"
-                                className="u-margin-bottom-sm"
-                                size="body-sm"
-                              >
-                                Project name
-                              </Heading>
-                              <div className="fpo u-margin-none">12 days</div>
-                            </CardBody>
-                          </Card>
-                        </GridItem>
-                      </Grid>
-                    </ProjectColumn>
-                  </OverflowListItem>
-                  <OverflowListItem>
-                    <ProjectColumn>
+                    </DragDropContainerHeader>
+                  ),
+                },
+                'container-2': {
+                  itemIds: [],
+                  emptyContent: (
+                    <>
+                      <TextPassage className="u-margin-bottom-lg">
+                        <p>
+                          Drag in available projects to build your course plan
+                        </p>
+                      </TextPassage>
+                      <svg
```

Comment:
Is this the illustration? Could you move it out of the jsx, either to the top of the page or another file, so it's easier to read the rest of the code?